### PR TITLE
IGAPP-464: Add e2e-test for content search

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: docs/e2e-tests.md [hunk 1]
```diff
@@ -72,6 +72,24 @@ Add the accessibility-id to a React component using `testID('Example-Component')
...
+* Start Server
+* Click `search icon`
+* Add to `Desired Capabilities`: 
+```
+{
+  "app": "<your-path-to>/integreat-app/native/android/app/build/outputs/apk/debug/app-debug.apk",
+  "platformName": "Android"
+  } 
+```
+* Click `start sesscion`
```
Comment (klinzo) [inline docs/e2e-tests.md:89]:
```suggestion
* Click `start session`
```

File: docs/e2e-tests.md [hunk 2]
```diff
@@ -72,6 +72,24 @@ Add the accessibility-id to a React component using `testID('Example-Component')
...
+* Now you can open a page in the emulator and inspect it
+
```
Comment (klinzo) [inline docs/e2e-tests.md:91]:
```suggestion
```

File: e2e-tests/native/test/specs/navigateToSearchResult.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import DashboardPage from '../pageobjects/dashboard.page'
+import SearchPage from '../pageobjects/search.page'
+import { navigateToDashboard } from '../utils/navigationUtils'
+import { contentSearch } from '../../../shared/constants'
+import { Selector } from '../Selector'
+
+describe('navigateToSearchResult', () => {
+  it('open and test content search', async () => {
```
Comment (klinzo) [inline e2e-tests/native/test/specs/navigateToSearchResult.e2e.ts:8]:
```suggestion
  it('should open and search content', async () => {
```

**Review by klinzo — APPROVED** [review]
I really like the simplicity of those tests :heart_eyes: On our way to the OKR :rocket:

File: docs/e2e-tests.md [hunk 4]
```diff
@@ -72,6 +72,24 @@ Add the accessibility-id to a React component using `testID('Example-Component')
 
 For more complex queries you should add/use a custom [Selector]('../e2e/native/Selector.ts) using [predicate strings](https://github.com/facebookarchive/WebDriverAgent/wiki/Predicate-Queries-Construction-Rules) for iOS and [UiSelectors](https://developer.android.com/reference/androidx/test/uiautomator/UiSelector) for Android.
 
+#### Using Appium
```
Comment (maxammann) [inline docs/e2e-tests.md:75]: I think this belongs to `native/docs/e2e-testing`. There is also already a section about Appiu .

**Review by maxammann — APPROVED** [review]

File: docs/e2e-tests.md (hunk 4 above)
Comment (SteffiStuffel) [reply docs/e2e-tests.md:75]: Good point. Moved it.

**Pull request merged**